daffodil-schematron/src/test/resources/xsd/embedded-1.dfdl.xsd

```diff
@@ -18,7 +18,8 @@
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
            xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/">
 
-    <xs:include schemaLocation="org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
+    <!-- schemaLocation can have upward relative path step, or can be from a classpath root. -->
+    <xs:include schemaLocation="../org/apache/daffodil/xsd/DFDLGeneralFormat.dfdl.xsd"/>
```

Comment:
This raises an interesting case. We use this DFDLGeneralFormat include in lots of schemas and the path is wrong for all of them if jars are extracted. Other resolves would expect the `org` directory to be in the same directory as the including schema, which id never the case.

If we go with my above suggestion, then at least everything will be broken for both Daffodil and things not using Daffodils resolver, so we're at least consistent. But that would mean every one of these DFDLGeneralFormat includes needs to be changed to have the right amount of up-steps, which is a pretty-invasive change. I guess we could special case imports starting with `org/apache/dafodil` and create a warning to suggest fixing this includes to use up-steps, with eventual deprecation.

Reply:
Yup. Well nothing will break, as it will try to use the path as is, then try it without the upsteps. 

I think we might eventually want the deprecation thing on paths without the up-steps, but I'm not sure. An absolute package-qualified name is awfully simple to use. Getting the relative paths right is only needed for this cybersecurity use case where people are separating the parse/unparse from the XSD validator. 

Having the up-steps on the schemaLocations makes refactoring the schema, e..g, changing locations of things harder. I would claim if a schema is only needed for parse/unparse, then one would prefer NOT to have those up-steps on the paths.